src/export/InstanceExporter.ts

```diff
@@ -26,7 +28,7 @@ export class InstanceExporter implements Fishable {
     instanceDef: InstanceDefinition,
     instanceOfStructureDefinition: StructureDefinition
   ): InstanceDefinition {
-    let rules = fshInstanceDef.rules.map(r => cloneDeep(r));
+    let rules = fshInstanceDef.rules.map(r => cloneDeep(r)) as FixedValueRule[];
```

Comment:
Same comment applies here as I left on `MappingExporter.ts` line 43: I'd prefer that we not _assume_ that all `insert` rules have already been filtered out.  See my comment on `MappingExporter.ts` for more discussion.